rescript/plugin_setup.py

```diff
@@ -464,6 +466,50 @@
 )
 
 
+plugin.methods.register_function(
+    function=replace_taxonomy,
+    inputs={
+        'taxonomy': FeatureData[Taxonomy]
+    },
+    parameters={
+        'taxonomy_replacement_map': MetadataColumn[Categorical],
+        'num_expected_ranks': Int % Range(1, None),
+    },
+    outputs=[('edited_taxonomy', FeatureData[Taxonomy])],
+    input_descriptions={
+        'taxonomy': 'Taxonomy strings data to be edited.'
+    },
+    parameter_descriptions={
+        'taxonomy_replacement_map': 'A tab-delimitad metadata file in which '
```

Comment:
maybe just call this "replacement_map" or something shorter to type?

Also I recommend dropping the example — as noted below, a tutorial would be a better place to communicate that specific example. (That example is also a bit controversial)